tests/installation/iscsi_configuration.pm

```diff
@@ -31,7 +31,7 @@ sub run {
     assert_screen 'iscsi-ibft';
     send_key 'alt-o';    # OK
     assert_screen 'disk-activation-iscsi';
-    wipe_iscsi_disk;    # At this point should be mounted and can proceed to erase it
+    # wipe_iscsi_disk;    # At this point should be mounted and can proceed to erase it
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
This is a separate commit, to be removed before I remove the WIP label. I assumed since this PR is still WIP with no VR it would not be an issue

Reply:
Makes sense though, thanks for bringing it up